src/app/components/course-item/course-item.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+export interface Course {
+  id: number;
+  title: string;
+  creationDate: string;
+  duration: number;
+  description: string;
+}
+
+@Component({
+  selector: 'app-course-item',
+  templateUrl: './course-item.component.html',
+  styleUrls: ['./course-item.component.scss'],
+})
+export class CourseItemComponent implements OnInit {
+  @Input()
+  public course?: Course;
+
+  @Output() newItemEvent = new EventEmitter<string | number>();
+
+  onDelete(value?: string | number) {
+    this.newItemEvent.emit(value);
```

Comment:
Like a code quality suggestion:  better to use the name same what exactly this method or eventEmitter will do. In this case let's rename like courseDelete to be consistent all code names